readme.md

```diff
@@ -22,8 +22,8 @@ Hold `Alt` key (`Option` on MacOS) to measure distances in Figma.
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Julia-Pushko.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Julia-Pushko.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="1021" alt="image" src="https://github.com/user-attachments/assets/f8400eba-5cad-48f0-9d10-46199ccd8437" />